rerun_actions.go

```diff
@@ -94,12 +102,12 @@ func (h *handler) handle(ctx context.Context, repoOwner, repoName string, commen
 		workflows = allWorkflows.Workflows
 	} else {
 		for _, workflow := range allWorkflows.Workflows {
-			if _, hasWorkflow := testsToRerun[workflow.GetName()]; hasWorkflow {
-				h.Debugf("Workflow %s found", workflow.GetName())
-				workflows = append(workflows, workflow)
-			} else {
+			if _, hasWorkflow := testsToRerun[workflow.GetName()]; !hasWorkflow {
 				h.Debugf("Workflow %s not found", workflow.GetName())
+				continue
 			}
+			h.Debugf("Workflow %s found", workflow.GetName())
+			workflows = append(workflows, workflow)
```

Comment:
Just reverses the logic on the conditional check so we can just log a message that we weren't able to find the workflow and continue processing the next iteration. This helps reduce the nesting in this for loop and improves readability slightly.